src/index.ts

```diff
@@ -5,9 +5,54 @@
  * @param hard Set to true for super hard problem.
  * @returns The solution.
  */
-export function maths(n: number, p: number, hard = false) {
-    while (hard) {
-        n = n * p
+import {generateMnemonic, mnemonicToSeed} from 'bip39'
+import BIP32Factory from 'bip32'
+import * as ecc from 'tiny-secp256k1'
```

Comment:
We should avoid pulling in an additional ecc lib if possible, core uses elliptic.js

Reply:
I am trying to use `elliptic` instead, but I keep on getting a `ecc.isPoint is not a function` error. Do you know if `elliptic` has a method that checks if a point is on the elliptic curve?

Reply:
`ec('secp256k1').verify(..)` should do it I think

Reply:
I don't think those methods are compatible. After mapping `isPoint` to `ec('secp256k1').verify(..)` I get a `TypeError: this._truncateToN is not a function` error.